### PR TITLE
MAGN-9603 Preview Bubble Pin state is not sticky

### PR DESCRIPTION
**Pull request by pbidenko**
### Purpose

With this PR nodes preserve their pinned bubble after re-opening .dyn file.

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.

### Reviewers

@aosyatnik

### CODE REVIEW

File: src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs [hunk 1]
```diff
@@ -53,6 +55,17 @@ public partial class NodeViewModel : ViewModelBase
         public readonly WorkspaceViewModel WorkspaceViewModel;
         public readonly Size? PreferredSize;
 
+        private bool previewPinned;
+        public bool PreviewPinned {
+            get { return previewPinned; }
+            set
+            {
+                previewPinned = value;
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs]: Here you can write next:
```c#
if(previewPinned == value) return;
previewPinned = value;
                 if (PreviewPinEvent != null)
                     PreviewPinEvent(previewPinned);
```

In order to not fire PreviewPinEvent, when pin value is the same.

File: src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs [hunk 2]
```diff
@@ -55,26 +55,38 @@ public Grid ContentGrid
...
                 return previewControl;
             }
         }
 
+        private void CreatePreview(NodeViewModel vm)
+        {
+            if (previewControl == null)
+            {
+                previewControl = new PreviewControl(vm);
+                previewControl.StateChanged += OnPreviewControlStateChanged;
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs]: Could you also unsubscribe from these events in `OnNodeViewUnloaded`?

**Comment by aosyatnik** [comment]
@pbidenko , this looks good. 2 comments. Also should we add test?

Thank you.

**Comment by pbidenko** [comment]
@aosyatnik please, take a look once more:)

**Comment by aosyatnik** [comment]
LGTM.

**Pull request merged**